src/functionaltests/resources/subscriptionRepeatHandlerOneMatch.json

```diff
@@ -0,0 +1,25 @@
+{
+  "created": "2017-07-26",
+  "notificationMeta": "http://127.0.0.1:3000/ei/test_subscription_rest",
+  "notificationType": "REST_POST",
+  "restPostBodyMediaType": "application/x-www-form-urlencoded",
+  "notificationMessageKeyValues": [
+    {
+      "formkey": "e",
+      "formvalue": "{parameter: [{ name: 'jsonparams', value : to_string(@) }, { name: 'runpipeline', value : 'mybuildstep' }]}"
+    }
+  ],
+  "repeat": false,
+  "requirements": [
+    {
+      "conditions": [
+        {
+          "jmespath": "ongoing==true"
+        }
+      ],
```

Comment:
What is the difference between these two json files?

Reply:
difference in number of conditions, because in one case aggragatedObjId should match one in second twice